microsoft/react-native-macos
# Remove incorrect menu display fallback

We can always just fallback to using the mouse location.

#### Please select one of the following
- [X ] I am making a fix / change for the macOS implementation of react-native

## Summary

When launching a menu, if the manager fails to find an event associated with the view then it enters an invalid fallback scenario for where to display the menu. We shouldn't ever launch the menu in the middle of the view.

It's more correct to fallback to the mouse location, which mirrors the actual intent of the presenter.

There could be an issue here if an SDX puts a control that drops a big menu too low then we'd redbox and go off the screen. That would require more refactoring and probably some of the positioning logic from callouts. Let's wait until we actually find a reason to update that behavior.

## Test Plan

Enabled both the fallback scenario and the default scenario for test runs in an internal test app and RNTester. Both show menus where expected. With this removal of an erroneous fallback, we no longer can pop up menus in unexpected locations.

RNTester
- Launch a menu without an anchorView and see it appears at the mouse location
- Launch a menu with an anchorView and see it appears below the anchorView (left aligned)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/361)
